src/components/LanguageFlyout/style.css

```diff
@@ -16,6 +16,7 @@
   min-width: 90px;
```

Comment:
Please merge dev into your branch and add `@import url('../Common/Colors.css');` at the top of this file.

Reply:
Done!